plugins/bulk-import/src/components/BulkImportPage.tsx

```diff
@@ -27,6 +28,7 @@ export const BulkImportPage = () => {
     excludedRepositories: {},
     approvalTool: ApprovalTool.Git,
   };
+  const queryClient = new QueryClient();
```

Comment:
When you move the `queryClient` const above `export const BulkImportPage`, then it will reuse the same instance also when the user switch to another page and come back later.

Both has pro and con. Its up to you.

But when you keep it here, you should ensure that it says the same instance also when rerendering this component. You should save it then in a useState or useRef.